README.md

```diff
@@ -80,86 +80,17 @@ size = "small" | "medium" | "large" | [( "2" | "4" | "8" )] , "xlarge";
 
 # Series
 
-.                           |  C    |  CO   |  CX   |  GN   |  M    |  N
+.                           |  CX   |  GN   |  M    |  N    |  U    |  UO
 ----------------------------|-------|-------|-------|-------|-------|------
-*Has GPUs*                  |       |       |       |  ✓    |       |
-*Hugepages*                 |       |       |       |       |  ✓    |  ✓
-*Cache backed RAM*          |       |  ✓    |       |       |       |
-*Dedicated CPU performance* |       |       |  ✓    |       |       |
-*Burstable CPU performance* |  ✓    |  ✓    |       |  ✓    |  ✓    |  ✓
-*Isolated emulator threads* |       |       |  ✓    |       |       |
-*vNUMA*                     |       |       |  ✓    |       |       |
+*Has GPUs*                  |       |  ✓    |       |       |       |
+*Hugepages*                 |       |       |  ✓    |  ✓    |       |
+*Cache backed RAM*          |       |       |       |       |       |  ✓
+*Dedicated CPU performance* |  ✓    |       |       |       |       |
+*Burstable CPU performance* |       |  ✓    |  ✓    |  ✓    |  ✓    |  ✓
+*Isolated emulator threads* |  ✓    |       |       |       |       |
+*vNUMA*                     |  ✓    |       |       |       |       |
 *vCPU-To-Memory Ratio*      |  1:2  |  1:4  |  1:4  |  1:4  |  1:4  |  1:8
 
-## C Series
-
-The C Series is quite neutral and provides resources for
-general purpose applications.
-
-*C* is the abbreviation for "Compute", hinting at the fact that
-this family is providing general compute resources to
-workloads.
-
-VMs of instance types will share physical CPU cores on a
-time-slice basis with other VMs.
-
-### C Series Characteristics
-
-Specific characteristics of this series are:
-- *Burstable CPU performance* - The workload has a baseline compute
-  performance but is permitted to burst beyond this baseline, if
-  excess compute is available
-- *vCPU-To-Memory Ratio (1:2)* - A vCPU-to-Memory ratio of 1:2
-
-### C Series Instance Types
-
-The following instance types are available in this series:
-
-Name       | Cores | Memory
------------|-------|-------
-c1.medium  | 1     | 2Gi
-c1.large   | 2     | 4Gi
-c1.2xlarge | 8     | 16Gi
-c1.4xlarge | 16    | 32Gi
-c1.8xlarge | 32    | 64Gi
-
-
-## CO Series
-
-The CO Series is based on the C Series, with the difference
-of being memory oversubscribed.
-
-*CO* is the abbreviation for "Compute and Oversubscribed"
-hinting at the neutral attitude towards workloads and the fact
-that instances of this type are memory oversubscribed.
-
-VMs of instance types will share physical CPU cores on a
-time-slice basis with other VMs.
-
-### CO Series Characteristics
-
-Specific characteristics of this series are:
-- *Burstable CPU performance* - The workload has a baseline compute
-  performance but is permitted to burst beyond this baseline, if
-  excess compute is available
-- *Cache backed RAM* - VM RAM is cached based in order to provide memory
```

Comment:
- will leverage file system cache and be fast, but as soon as node memory arises, the memory of the VMs will be pushed out of RAM, because the cache will be dropped.